callouts/tests/load-testing/run-test.sh

```
@@ -0,0 +1,755 @@
+    # Get test parameters
+    TEST_DURATION="${duration:-$(echo "$test_mode_config" | jq -r '.duration')}"
+    TEST_VUS="${vus:-$(echo "$test_mode_config" | jq -r '.vus')}"
+    WARMUP_DURATION="${warmup:-$(echo "$test_mode_config" | jq -r '.warmup // "0s"')}"
+
+    local container_port=$(echo "$service_config" | jq -r '.container_port // .port // "8080"')
+    local health_check_container_port=$(echo "$service_config" | jq -r '.health_check_container_port // "80"')
+
+    print_section "Load Test: $service_type/$test_mode/$scenario"
+    log_info "Duration: $TEST_DURATION, VUs: $TEST_VUS, Warmup: $WARMUP_DURATION"
+
+    local timestamp=$(date +%Y%m%d_%H%M%S)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)